resources/views/print/send.blade.php

```diff
@@ -0,0 +1,15 @@
+<div class="card">
+    <div class="card-header">@lang('print.transfer_money')</div>
+    <div class="card-body">
+        <form method="POST" action="{{ route('print.send') }}">
```

Comment:
**send** is not descriptive enough, just look 2 lines above - that is descriptive.